papers/ReTraCk/README.md

```diff
@@ -174,9 +171,51 @@ python ./tests/debug/launch_schema_retriever.py
 ```
 Inference will generate Class and Relation predictions, along with merged results, and print out Recall numbers for both predictions types.
 
-### Use ReTraCkRetriever in your codebase
```

Comment:
I think we should be this piece before going into the training instructions:

### Use ReTraCkRetriever in your codebase

For examples on how to use ReTraCkRetriever in your code base, please check /tests/debug/launch_schema_retriever.py